### ISSUE TITLE
Conditional reveals with an input not being announced on JAWS

### ISSUE THREAD

**Issue by charge-valtech** [alphagov/govuk-frontend#935]
This is feedback from a JAWS user that tested our prototype. I don't currently have access to JAWS so have been unable to test myself.

"This screen has 3 radio buttons, all are easy to navigate to and select. The radio button someone else when selected expands, the end user has to either tab or down arrow until JAWS announces what relationship are you to the applicant. There is no immediate audio to let the end user no that an edit field has visually appeared on the screen. I decided to ignore selecting a radio button and activated the continue button"

Browser IE 11
JAWS V17

**Comment by nickcolley** [comment]
See https://github.com/alphagov/govuk-frontend/issues/979#issuecomment-530320650 for investigation into aria attributes on conditional reveals.

**Comment by adamliptrot-oc** [comment]
With JAWS 18 (I don’t have 17 to test with) on IE11 the aria works as expected (but you have to use enter rather than space to action the radio/checkbox in order for it to fire the aria). When the reveal opens you get a prompt to move to the controlled element.

**Comment by 36degrees** [comment]
Consolidating into #1972.
